### PR TITLE
Update 60-sensor.hwdb

### PR DESCRIPTION
**Pull request by Luxiter**
Fix Asus T300FA rotation matrix

### CODE REVIEW

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -182,6 +182,9 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svn*ASUSTeK*:*pn*TP412UA:*
 sensor:modalias:acpi:BOSC0200*:dmi:*svn*ASUSTeK*:pn*BR1100FKA:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, -1
 
+sensor:modalias:acpi:BMA250E*:dmi:*:svnASUSTeK*:pnT300FA:*
+ ACCEL_MOUNT_MATRIX=-0,1,0;1,0,0;0,0,1
```
Comment (yuwata) [inline hwdb.d/60-sensor.hwdb]: `-0` -> `0` ??

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/60-sensor.hwdb (hunk 1 above)
Comment (yuwata) [reply hwdb.d/60-sensor.hwdb]: please also insert a space like the other entries.

Comment (Luxiter) [reply hwdb.d/60-sensor.hwdb]: a typo sorry 
the correct code 
```
sensor:modalias:acpi:BMA250E*:dmi:*:svnASUSTeK*:pnT300FA:*
 ACCEL_MOUNT_MATRIX=0,1,0;1,0,0;0,0,1
```

**Review by Luxiter — COMMENTED** [review]
correct a typo error

**Comment by yuwata** [comment]
Please also insert spaces.

**Pull request merged**